client/app/layouts/_application.sass

```diff
@@ -1,3 +1,15 @@
+// Product overrides of Boostrap & Patternfly variables
```

Comment:
on snap, we DONT have a file for all misc vendor overrides. this needs to happen, not in this commit but **thank you** for mentioning this as an override here